file-formats/evtb/evtb-v1.json

```diff
@@ -0,0 +1,166 @@
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/evtb/evtb-v1.json",
+  "title": "General Information",
+  "type": "object",
+  "properties": {
+    "formatVersion": {
+      "title": "ABAP File Format Version",
+      "description": "The ABAP file format version",
+      "type": "string",
+      "const": "1"
+    },
+    "header": {
+      "title": "Header",
+      "description": "Header",
+      "type": "object",
+      "properties": {
+        "description": {
+          "title": "Description",
+          "description": "Description of the ABAP object",
+          "type": "string",
+          "maxLength": 60
+        },
+        "originalLanguage": {
+          "title": "Original Language",
+          "description": "Original language of the ABAP object",
+          "type": "string",
+          "minLength": 2,
+          "maxLength": 2,
+          "pattern": "^[a-z]+$"
+        },
+        "abapLanguageVersion": {
+          "title": "ABAP Language Version",
+          "description": "ABAP language version",
+          "type": "string",
+          "enum": [
+            "standard",
+            "keyUser",
+            "cloudDevelopment"
+          ],
+          "enumTitles": [
+            "Standard",
+            "ABAP for Key Users",
+            "ABAP Cloud Development"
+          ],
+          "enumDescriptions": [
+            "Standard",
+            "ABAP for key user extensibility",
+            "ABAP cloud development"
+          ],
+          "default": "standard"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "description",
+        "originalLanguage"
+      ]
+    },
+    "producerNamespace": {
+      "title": "Namespace",
+      "description": "Namespace",
+      "type": "string",
+      "maxLength": 15
+    },
+    "producerLegacy": {
+      "title": "Producer Legacy",
+      "description": "Producer legacy",
+      "type": "boolean"
+    },
+    "producer": {
+      "title": "Producer",
+      "description": "Producer",
+      "type": "string",
+      "maxLength": 36
+    },
+    "producerVersion": {
+      "title": "Producer Version",
+      "description": "Producer version",
+      "type": "string",
+      "maxLength": 4,
+      "pattern": "^[0-9]+$"
+    },
+    "boName": {
+      "title": "Business Object",
+      "description": "Business object",
+      "type": "string",
+      "maxLength": 30
+    },
+    "boOperation": {
+      "title": "Business Object Operation",
+      "description": "Business object operation",
+      "type": "string",
+      "maxLength": 18
+    },
+    "producerTopic": {
+      "title": "Type",
+      "description": "Type",
+      "type": "string",
+      "maxLength": 83
+    },
+    "events": {
+      "title": "Events",
+      "type": "array",
+      "items": {
+        "title": "Event",
+        "type": "object",
+        "properties": {
+          "eventVersion": {
```

Comment:
does it make sense to group the 3 version fields?
versions {
normal,
minor,
patch
}

Reply:
All events shall be shown as flat-strucured list. Therefore, we won't create a sub structure.